apps/admin-gui/src/app/vos/components/application-form-list/application-form-list.component.ts

```diff
@@ -57,6 +57,44 @@ export class ApplicationFormListComponent implements OnChanges {
 
   }
 
+  disabledDependency(item: ApplicationFormItem): String {
+    let mess: String;
+    let dep: String;
+    switch (item.disabled) {
+      case 'ALWAYS':
+        return "";
+      case 'IF_PREFILLED':
+        mess = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.IF_PREFILLED');
+        dep = this.applicationFormItems.find(i => i.id === item.disabledDependencyItemId).shortname;
```

Comment:
This will throw an error if the dependency is not specified. The find will return `undefined` it cannot access `.shortname` on it. The same applies for the other use-cases.